services/ui-src/src/components/layout/Footer.tsx

```diff
@@ -105,57 +114,93 @@ const sx = {
     },
   },
   footerTopFlex: {
-    minH: "12.5rem",
-    justifyContent: "space-between",
-    alignItems: "flex-start",
-    paddingY: "2rem",
-    ".mobile &": {
-      flexDirection: "column",
-      alignItems: "center",
-    },
+    flexWrap: "wrap",
   },
-  footerTopLogoFlex: {
-    paddingRight: "1rem",
-    ".mobile &": {
-      minWidth: "200px",
-      paddingRight: 0,
-      justifyContent: "center",
+  footerTopLeftContainer: {
+    flex: "0 0 50%",
+    maxWidth: "100%",
+    minWidth: "100%",
```

Comment:
Question: Could we just use `width: "100%"` here? Same question with other container below.